internal/interfaces/grpc/permissions/permissions.go

```diff
@@ -300,6 +303,10 @@ func Whitelist() map[string][]bakery.Op {
 			Entity: EntityTrade,
 			Action: "write",
 		}},
+		fmt.Sprintf("/%s/ServerReflectionInfo", grpc_reflection_v1alpha.File_reflection_grpc_reflection_v1alpha_reflection_proto.Services().Get(0).FullName()): {{
+			Entity: Reflection,
+			Action: "write",
```

Comment:
Action should be read?
```suggestion
			Action: "read",
```

Reply:
Done